public/app/features/logs/components/panel/LogLineDetails.tsx

```diff
@@ -193,19 +196,30 @@ InlineLogLineDetails.displayName = 'InlineLogLineDetails';
 
 export const LOG_LINE_DETAILS_HEIGHT = 35;
 
-const getStyles = (theme: GrafanaTheme2, mode: LogLineDetailsMode) => ({
+const getStyles = (theme: GrafanaTheme2, mode: LogLineDetailsMode, showControls?: boolean) => ({
   inlineWrapper: css({
     gridColumn: '1 / -1',
     height: `${LOG_LINE_DETAILS_HEIGHT}vh`,
     padding: theme.spacing(1, 2, 1.5, 2),
     marginRight: 1,
   }),
-  container: css({
-    overflow: 'auto',
+  inlineContainer: css({
+    backgroundColor: theme.colors.background.elevated,
+    border: `1px solid ${theme.colors.border.weak}`,
+    borderRadius: theme.shape.radius.default,
+    boxShadow: theme.shadows.z2,
     height: '100%',
-    boxShadow: theme.shadows.z1,
+    overflow: 'auto',
+  }),
+  container: css({
+    backgroundColor: theme.colors.background.elevated,
     border: `1px solid ${theme.colors.border.medium}`,
-    borderRight: mode === 'sidebar' ? 'none' : undefined,
+    borderBottomRightRadius: showControls ? undefined : theme.shape.radius.default,
+    borderRight: mode === 'sidebar' && showControls ? 'none' : undefined,
+    borderTopRightRadius: showControls ? undefined : theme.shape.radius.default,
+    boxShadow: theme.shadows.z3,
+    height: '100%',
+    overflow: 'auto',
   }),
   scrollContainer: css({
```

Comment:
when it is inline it's not really elevated, so could use theme.colors.background.secondary or primary, and no shadow. 

border should always be `weak` in both inline and sidebar